### PR TITLE
Codejam 3 coreJS

### PR DESCRIPTION
**Pull request by pirsfrommi6**

### CODE REVIEW

**Comment by pirsfrommi6** [comment]
good PR

File: src/recursion.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+function recursion(arg) {
+	if (!(arg.left && arg.right)) {
+		return [ [ arg.value ] ];
+	}
+	// only if left&right not null & ===
+	const left = recursion(arg.left);
+	const right = recursion(arg.right);
+	const arr = [];
+	const leftsize = left.length;
```
Comment (SergeyOvsyanik) [inline src/recursion.js:9]: leftLength is better

File: src/recursion.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	arr.push([ arg.value ]);
```
Comment (SergeyOvsyanik) [inline src/recursion.js:10]: Please add some blank lines for improve code reading

File: src/recursion.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+function recursion(arg) {
```
Comment (SergeyOvsyanik) [inline src/recursion.js:1]: Please rename "arg" for more convenient name.

**Pull request merged**